tools/Data_explo/2_gala_EA.xml

```diff
@@ -0,0 +1,93 @@
+<tool id="tool_environment" name="Abundance in environment" version="0.0">
+    <description>Show the abundance according to the location</description>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <expand macro="Explo_requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
+            '$__tool_directory__/Exe2_EA.r'

```

Comment:
I would recommend renaming the scripts to something more meaningful to make them more maintainable over the long run.